src/demoserver.cpp

```diff
@@ -233,9 +233,39 @@ void DemoServer::handle_packet(AOPacket packet)
             QString packet = "CT#DEMO#" + tr("min_wait is deprecated. Use the client Settings for minimum wait instead!") + "#1#%";
             client_sock->write(packet.toUtf8());
         }
+        else if (contents[1].startsWith("/debug"))
+        {
+            QStringList args = contents[1].split(" ");
+            if (args.size() > 1)
+            {
+              bool ok;
+              int toggle = args.at(1).toInt(&ok);
+              if (ok && (toggle == 0 || toggle == 1)) {
+                debug_mode = toggle == 1;
+                QString packet = "CT#DEMO#" + tr("Setting debug mode to ") + (debug_mode ? "true" : "false") + "#1#%";
```

Comment:
pretty sure `.arg()` with booleans works

```suggestion
                QString packet = "CT#DEMO#" + tr("Setting debug mode to %1").arg(debug_mode) + "#1#%";
```